src/Workspaces/CoreTest/ChecksumTests.cs

```diff
@@ -19,7 +19,7 @@ public void ValidateChecksumFromSpanSameAsChecksumFromBytes1()
             var checksumA = Checksum.Create(checksum1, checksum2);
 
             // Running this test on multiple target frameworks with the same expectation ensures the results match
-            Assert.Equal(Checksum.FromBase64String("N30m5jwVeMZzWpy9cbQbtSYHoXU="), checksumA);
+            Assert.Equal(Checksum.FromBase64String("RRbspG+E4ziBC5hOWyrfCQ=="), checksumA);
```

Comment:
expected change.  we are legitimately using a different hash.  

the tests *do* check though that we get the same result on netfx vs netcore (that way we can have roslyn communicating across process while still agreeing on the results).